libs/backend-apisix/src/typing.ts

```diff
@@ -108,7 +108,7 @@ export interface StreamRoute {
   id: string;
   desc?: string;
   labels?: Labels;
-  name: string;
+  //name: string; // As of 3.9.1, APISIX does not support name on the stream route
```

Comment:
Should we remove this unused value?

Reply:
No, I will add it to APISIX. We can use it later.